### PR TITLE
Fixes #9669 - Attach/Remove sub actions for host

### PR DESCRIPTION
**Pull request by parthaa**
This commit adds functionality for the following calls
hammer host subscription attach --subscription-id=1000 --quantity=333 --host-id=1000
hammer host subscription remove --subscription-id=1000 --host-id=1000
hammer host subscription auto-attach --host-id=1000

### CODE REVIEW

**Comment by parthaa** [comment]
requires https://github.com/Katello/katello/pull/6233

File: lib/hammer_cli_katello/host_subscription.rb [hunk 1]
```diff
@@ -27,6 +27,48 @@ class CreateCommand < HammerCLIKatello::SingleResourceCommand
       end
     end
 
+    class AddSubscriptions < HammerCLIKatello::SingleResourceCommand
+      resource :host_subscriptions, :add_subscriptions
+      command_name "attach"
+      success_message _("subscription attached to the host successfully")
```
Comment (akofink) [inline lib/hammer_cli_katello/host_subscription.rb]: s/subscription/Subscription/

**Comment by akofink** [comment]
@parthaa Please add tests for these three sub-commands.

**Comment by parthaa** [comment]
@akofink updated.

**Comment by akofink** [comment]
Tests look good. Works as expected. ACK

**Pull request merged**